### PR TITLE
Update containerd to 564600ee79aefb0f24cbcecc90d4388bd0ea59de.

### PR DESCRIPTION
**Pull request by Random-Liu**
Update containerd to 564600ee79aefb0f24cbcecc90d4388bd0ea59de to pick up https://github.com/containerd/containerd/pull/1707, which could potentially get rid of the deadlock we are hitting in node e2e test.

Signed-off-by: Lantao Liu <lantaol@google.com>

### CODE REVIEW

File: hack/update-vendor.sh [hunk 1]
```diff
@@ -24,7 +24,12 @@ set -o pipefail
 ROOT="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"/..
 cd ${ROOT}
 
+echo "Update vendor"
+vndr
```
[the remark this answers is not in the document]
Comment (Random-Liu) [reply hack/update-vendor.sh]: What do you usually use? I just use `vndr` usually.

[the remark this answers is not in the document]
Comment (mikebrow) [reply hack/update-vendor.sh]: if I'm adding just one I'll use vndr package.. it runs a hundred times faster that way

example:
```
mike@mike-VirtualBox:~/go/src/github.com/kubernetes-incubator/cri-containerd$ sudo vndr github.com/renstrom/dedent
2017/11/02 16:31:13 Collecting initial packages
2017/11/02 16:31:13 Download dependencies
2017/11/02 16:31:15 	Clone github.com/renstrom/dedent, revision 020d11c3b9c0c7a3c2efcc8e5cf5b9ef7bcea21f
2017/11/02 16:31:16 	Finished clone github.com/renstrom/dedent
2017/11/02 16:31:16 Dependencies downloaded. Download time: 542.317315ms
2017/11/02 16:31:16 Collecting all dependencies
2017/11/02 16:31:17 Clean vendor dir from unused packages
2017/11/02 16:31:17 Success
2017/11/02 16:31:17 Running time: 4.261910138s
``` 
vs 
2017/11/02 14:46:55 Running time: 6m43.596822949s for a full vndr

[the remark this answers is not in the document]
Comment (Random-Liu) [reply hack/update-vendor.sh]: Hm. More complex than I thought.
Will remove the update in the `hack/update-vendor.sh` then. Seems like a overkill to wrap everything. :p

**Comment by Random-Liu** [comment]
@mikebrow Done.

**Pull request merged**